types/wordpress__blocks/index.d.ts

```diff
@@ -27,7 +27,6 @@ export type CSSDirection = 'top' | 'right' | 'bottom' | 'left';
 export type BlockAlignment = 'left' | 'center' | 'right' | 'wide' | 'full';
 
 export interface BlockEditProps<T extends Record<string, any>> extends BlockSaveProps<T> {
-    readonly className: string;
```

Comment:
looks like this is appropriate to add to the save props, but isn't it also available on `edit`? as in, is it accurate to remove from here?

Reply:
Since BlockEditProps extends BlockSaveProps both will have `className`. I just thought it made sense to store the property in BlockSaveProps since thats being used as the base interface to extend from.